### PR TITLE
WCAG SC 2.4.13 - Testing Script for page break

### PR DESCRIPTION
**Pull request by Ricokola**
Based on the change log of WCAG 2.2: [https://www.w3.org/TR/WCAG22/#changelog](https://www.w3.org/TR/WCAG22/#changelog), This success criterion has been removed on July 15 2022

Creation of new JS file (wcag.js) for testing Web pages against certain identified success criteria.

### CODE REVIEW

**Review by duboisp — CHANGES_REQUESTED** [review]
* [x] Please wrap the code to ensure there is no interference with any other JS code.
* [x] Modify the build script to include this file.
* [x] We need to run this code only after WET has been ready and all other WET component has been initialized and marked ready.

File: src/core/wcag.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ */
+
+var $pgbrk = $( ".pg-brk-aft" ),
+	msg = "WCAG SC 2.4.13 - The page break (.pg-brk-aft) is not ";
+
+if ( $pgbrk.length > 0 ) {
+	$pgbrk.each( function() {
+		var id = $( this ).next().attr( "id" );
+		if ( id ) {
+			if ( !$( "a[href*=" + id + "]" ).length ) {
```
Comment (duboisp) [inline src/core/wcag.js:16]: We will need to change the CSS attribute operator

**Review by duboisp — CHANGES_REQUESTED** [review]

**Comment by GormFrank** [comment]
@Ricokola Please rebase to upstream/master and @duboisp Have your requested changes been fixed / is it good to go?

**Comment by GormFrank** [comment]
Concerned about it running every page load to look for page breaks when 99% of all pages using WET don't use page breaks.

Maybe a different approach could be done to programmatically load the script instead of having the check for class. Something  "softer" in terms of performance (even though we are not talking about a big hit anyway).

CC:  @duboisp @Ricokola

**Review by duboisp — COMMENTED** [review]
- [ ] Can you add a link, in comments, of a valid and invalid page with page-break. The invalid page should enumerate all the use case.
- [x] Add un delay de 250 milisecondes (setTimeout) pour ne pas interférer avec les autres plugins. 
- [ ] Add a blurb dans notre page de documentation dans les utilities.

**Comment by Ricokola** [comment]
Based on the change log of WCAG 2.2: [https://www.w3.org/TR/WCAG22/#changelog](https://www.w3.org/TR/WCAG22/#changelog), This success criterion has been removed on July 15 2022

**Pull request closed without merging**